packages/docs/docs/alternative-getting-started.mdx

```diff
@@ -0,0 +1,131 @@
+---
+sidebar_position: 9
+id: alternative-getting-started
+---
+
+#  Getting started with create eth-app
+
+## Installation
+
+You’ll need to have Node 14 or later version on your local development machine (but it’s not required on the server). You can use nvm (macOS/Linux) or nvm-windows to switch Node versions between different projects.
+
+You'll also need Yarn on your local development machine. This is because Create Eth App relies on Yarn Workspaces, a feature not supported by Npm.
+
+To quick start development with useDapp run:
+
+```bash yarn
+yarn create eth-app my-eth-app
+```
+
+It creates folder my-eth-app and inside you have everything created to start.
+
+```
+my-eth-app
+├── README.md
+├── node_modules
+├── package.json
+├── .gitignore
+└── packages
+    ├── contracts
+    │   ├── README.json
+    │   ├── package.json
+    │   └── src
+    │       ├── abis
+    │       │   ├── erc20.json
+    │       │   └── ownable.json
+    │       ├── addresses.js
+    │       └── index.js
+    ├── react-app
+    │   ├── README.md
+    │   ├── package.json
+    │   ├── node_modules
+    │   ├── public
+    │   │   ├── favicon.ico
+    │   │   ├── index.html
+    │   │   └── manifest.json
+    │   └── src
+    │       ├── App.css
+    │       ├── App.js
+    │       ├── App.test.js
+    │       ├── ethereumLogo.svg
+    │       ├── index.css
+    │       ├── index.js
+    │       ├── serviceWorker.js
+    │       └── setupTests.js
+    └── subgraph
+        ├── README.md
+        ├── abis
+        │   └── erc20.json
+        ├── package.json
+        ├── schema.graphql
+        ├── src
+        │   └── mappings
+        │       ├── tokens.ts
+        │       └── transfers.ts
+        └── subgraph.yaml
+```
+
+## Start
+
+Just run
+
+```bash yarn
+yarn react-app:start
+```
+
+After that the app works, so open [http://localhost:3000/](http://localhost:3000/).
+
+## Frameworks
```

Comment:
UseDapp doesn't work with any other framework than the default React, so I propose we remove this section from our doc.

Reply:
Fine